packages/Webkul/Tax/src/Tax.php

```diff
@@ -151,7 +151,7 @@ public function isTaxApplicableInCurrentAddress($taxCategory, $address, $operati
         // dump($address);
         foreach ($taxRates as $rate) {
             if (
-                $address->state != '*'
+                $rate->state != '*'
```

Comment:
This change is not relevant. `$address->rate` was added because you can set the state to "*" in the "Default Destination Calculation" configuration. This address will be used for guests and customers without any saved addresses to calculate the taxes before saving the address in the cart.